heat/core/dndarray.py

```diff
@@ -1113,6 +1181,11 @@ def __getitem__(self, key):
                     key[self.split] = key[self.split] - chunk_start
                     arr = self.__array[tuple(key)]
                     gout = list(arr.shape)
+                elif key[self.split] < 0 and self.gshape[self.split] + key[self.split] in range(chunk_start, chunk_end):
```

Comment:
Is this a bug fix from another issue? If so, please reference it so that we can close it

Reply:
fixes #293